README.md

```diff
@@ -1,198 +1,66 @@
-# HUTCH
 
-HUTCH is an application that allows federated access to summary statistics of medical data at an institution.
+![Hutch](https://raw.githubusercontent.com/HDRUK/hutch/main/assets/Hutch%20splash%20bg.svg)
```

Comment:
Readme looks nice! Logo isn't showing for me though, same with the Associated Orgs at the bottom

Reply:
that's cos i'm linking to their raw source on `main` but they aren't on `main` until this PR merges ;)

It'll work post merge.